state-chain/pallets/cf-elections/src/mock.rs

```diff
@@ -64,6 +64,20 @@ impl TestSetup {
 	pub fn all_authorities(&self) -> Vec<u64> {
 		(0..self.num_contributing_authorities + self.num_non_contributing_authorities).collect()
 	}
+
+	pub fn contributing_authorities(&self) -> Vec<u64> {
+		self.all_authorities()
+			.into_iter()
+			.take(self.num_contributing_authorities as usize)
+			.collect()
+	}
+
+	pub fn non_contributing_authorities(&self) -> Vec<u64> {
+		self.all_authorities()
+			.into_iter()
+			.skip(self.num_contributing_authorities as usize)
+			.collect()
```

Comment:
could combine them and `split`? - means they're not going to go out of sync

Reply:
The intention is that this is just the test setup and therefore is immutable - it doesn't return the *current* contributing/non-contributing authorities.